### PR TITLE
IDVA6-368 remove authorise person success page

### PR DESCRIPTION
**Pull request by mmodzelewski-ch**
Link to Jira:
https://companieshouse.atlassian.net/browse/IDVA6-368

Changes:
- Manage authorised people page updated to display success banner on person removal
- Problems with debugger fixed
- Test updated and new added

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by bmchugh-ch — APPROVED** [review]

**Review by bmchugh-ch — DISMISSED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/routers/handlers/yourCompanies/manageAuthorisedPeopleHandler.ts [hunk 1]
```diff
@@ -61,4 +52,40 @@ export class ManageAuthorisedPeopleHandler extends GenericHandler {
...
+                this.viewData.cancelledPerson = cancellation.userEmail;
+            }
+        }
+    }
+
+    private async handleRemoval (removal: Removal) {
+        if (removal.removePerson === constants.CONFIRM) {
+            const isUserRemovedFromCompanyAssociations = (await removeUserFromCompanyAssociations(removal.userEmail, removal.companyNumber)) === constants.USER_REMOVED_FROM_COMPANY_ASSOCIATIONS;
+            if (isUserRemovedFromCompanyAssociations) {
+                this.viewData.removedPerson = removal.userName ? removal.userName : removal.userEmail;
```
Comment (kkonuganti-ch) [inline src/routers/handlers/yourCompanies/manageAuthorisedPeopleHandler.ts:69]: do we have something called userName? the only thing we have is display name. Display name by default is "Not Provided". Should we not check by user_id?

File: src/services/userCompanyAssociationService.ts [hunk 2]
```diff
@@ -217,6 +209,32 @@ export const addUserEmailAssociation = async (email: string, companyNumber: stri
         }
     }
 };
+export const getCompanyAssociations = async (companyNumber: string, cancellationOrRemoval: Cancellation | Removal | undefined): Promise<Associations> => {
+    // We will replace this hard coded value with the API call once the API is available
+
+    if (cancellationOrRemoval && "cancelPerson" in cancellationOrRemoval) {
+        return getCompanyAssociationsAfterCancellation(companyNumber, cancellationOrRemoval);
```
Comment (kkonuganti-ch) [inline src/services/userCompanyAssociationService.ts:216]: technically they both do the same thing. If cancelled or removed the status is set to Removed. Dont know if we need 2 separate calls here?

Comment (mmodzelewski-ch) [reply src/services/userCompanyAssociationService.ts:216]: similar as above

**Review by kkonuganti-ch — APPROVED** [review]

**Pull request merged**